experimental/externals.yaml

```diff
@@ -0,0 +1,7 @@
+repositories:
```

Comment:
This is what a "external" configuration would look like: aka where do we fetch resources from.
With the possibility to fetch only task, or pipelines or both (and later on.. more) and also ignoring versions (tags) if need be.